docs/atomic-sign.1.md

```diff
@@ -0,0 +1,55 @@
+% ATOMIC(1) Atomic Man Pages
+% Brent Baude
+% August 2016
+# NAME
+atomic-sign- Create a signature for a local image
+# SYNOPSIS
+**atomic sign**
+[**-h**|**--help**]
+
+[**-o**, **--output**]
+[**--sign_key**]
+[ image ... ]
+
+# DESCRIPTION
+**atomic sign** will create a local signature for one or more local images that have 
+been pulled from a registry. Unless overridden, the signature will end up in a directory
+as follows:
+
+ /var/lib/atomic/containers/registry_name/image_name@sha256:manifest/
+
+The signatures are named 'signature-INTEGER' where the integer is increased by 1 each
+time.
+
+The default storage location for signatures is /var/lib/atomic/containers.  A different
+default location can be defined in /etc/atomic.conf with the key **default-sigstore-path**.
+
+# OPTIONS
+**-h** **--help**
+  Print usage statement.
+
+**-o** **--output**
+  File path for storing the signature, overriding the default signature
+  location, /var/lib/atomic/containers.
+
+**--sign_key**
+  Override the default signer and use the defined signer instead. You can define
+  a default in /etc/atomic.conf with the key **default_signer**.
+
+
+# EXAMPLES
+Sign the busybox image from docker.io
+
+    atomic sign docker.io/busybox
+    
+Sign the busybox image and store the signature in /tmp
+
+    atomic sign -o /tmp/busybox.sig docker.io/busybox
+
+Sign the busybox image with the identify of foo@bar.com
+
+    atomic --sign-key foo@bar.com docker.io/busybox
+
```

Comment:
Per the **WARNING** suggested above, all of this should use `privateregistry.example.com` instead of `docker.io`.